src/test/java/bio/terra/common/fixtures/ResourceFixtures.java

```diff
@@ -17,4 +17,24 @@ public static GoogleProjectResource randomProjectResource(BillingProfileModel bi
             .profileId(UUID.fromString(billingProfile.getId()));
     }
 
+    public static GoogleProjectResource randomProjectResourceAndName(BillingProfileModel billingProfile) {
+
+        return new GoogleProjectResource()
+            .googleProjectId(ProfileFixtures.randomizeName("fake-test-project-id"))
+            .googleProjectNumber(shuffleString("123456"))
+            .profileId(UUID.fromString(billingProfile.getId()));
+    }
+
+    public static String shuffleString(String input) {
+        List<String> newProjectNum = Arrays.asList(input.split(""));
+        Collections.shuffle(newProjectNum);
```

Comment:
FWIW, you can just do `return StringUtils.join(newProjectNum, "")`

Reply:
Instead of the shuffle?

Reply:
Whoops, sorry, right, I meant: `return StringUtils.join(Collections.shuffle(newProjectNum), "")`

Reply:
Nice - Much cleaner! Thanks!